Add Parachain Parameters and More Context for Auctions (#3026)
w3f/polkadot-wiki: docs/maintain/maintain-polkadot-parameters.md

### Parachains

| Crowdloans and Auctions  | Time    | Slots   | Description                                                                      |
| ------------------------ | ------  | ------- | -------------------------------------------------------------------------------- |
| Auction duration         | 7 days  | 100_800 | The total duration of the slot auction, subject to the candle auction mechanism. |
| Opening period           | 2 days  | 28_800  | The opening period of the slot auction.                                          |
| Ending period            | 5 days  | 72_000  | The ending period of the slot auction.                                           |

| Parachain Slot           | Time     | Slots         | Description                                                                              |
| ------------------------ | -------- | ------------- | ---------------------------------------------------------------------------------------- | 

> | Slot period              | 12 weeks | 1_209_600     | The length of one lease period for a parachain slot.                                     |

Reviewer: Going with the terminology used on Polkadot JS - https://polkadot.js.org/apps/#/parachains/auctions

```suggestion
| Lease period             | 12 weeks | 1_209_600     | The length of one lease period in a parachain slot.                                     |
```

> | Total slot duration      | 2 years  | 10_512_000    | The length of the total lease period for a parachain slot, consisting of 8 slot periods. |

Reviewer: Consider changing the description to -

"The maximum duration a parachain can lease by winning a slot auction (8 lease periods)"

Author: ```suggestion
| Total slot duration      | 2 years  | 10_512_000    | The maximum duration a parachain can lease by winning a slot auction (8 lease periods). |
```
